assets/js/webpack-entry.js

```diff
@@ -6,129 +6,13 @@
 /*
     swap vue.min out for vue if you need vue devtools
 */
-const Vue = require("vue/dist/vue.min");
-
 const plugins = require("./plugins");
 const polyfills = require("./polyfills");
-const axios = require("axios");
-const moment = require("moment");
 
 /**
  * Initialize polyfils here.
  */
 
-// Plolyfills to add functionality to IE.
-polyfills.forEachPolyfill();
```

Comment:
we will still need this forEach polyfill because `.forEach` get used on a nodeList inside a function in the `plugins.js` module.